raiden/transfer/mediated_transfer/mediator.py

```diff
@@ -179,22 +179,22 @@ def filter_used_routes(transfers_pair, routes):
     return list(channelid_to_route.values())
 
 
-def get_payee_channel(channelidentifiers_to_channels, transfer_pair):
-    """ Returns the payee channel of a given transfer pair. """
+def get_payee_channel(
+        channelidentifiers_to_channels: typing.ChannelMap,
+        transfer_pair: MediationPairState,
+) -> typing.Optional[NettingChannelState]:
```

Comment:
Now both here and in the `payer_channel`, `None` may be returned. I am thinking right now if we should go through all the places in the file where `transfers_pair` are iterated and make sure to take into account the fact that a channel can now be None. 

In a transfer pair, the payer channel can be closed but the payee channel have a stuck transfer so it will need checking.

Reply:
> I am thinking right now if we should go through all the places in the file where transfers_pair are iterated and make sure to take into account the fact that a channel can now be None.

Probably, I think most of the loops assume both channels exist.

Reply:
Done with the latest commit